sensibo/syncHomeKitCache.js

```diff
@@ -218,9 +218,10 @@ module.exports = (platform) => {
 
 				case 'ClimateReact':
 					deviceExists = platform.devices.find(device => {
-						return device.id === accessory.context.deviceId && device.remoteCapabilities
+						device.id === accessory.context.deviceId && device.remoteCapabilities
```

Comment:
I think this still needs to return for the .find to work?
```suggestion
						return device.id === accessory.context.deviceId && device.remoteCapabilities
```